### PR TITLE
Input-field component

### PR DESCRIPTION
**Pull request by pflopez**
Added input-field component, stories on storybook and a story to demo usage on a form

Screen Shot 2024-05-15 at 11 27 46
Screen Shot 2024-05-15 at 12 04 28

### CODE REVIEW

File: src/interface/src/styleguide/input/input.stories.ts [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+ * ```
+ * // use with regular forms
+ * <sg-input-field>
+ *   <input sgInput placeholder="..."  value="...">
+ * </sg-input-field>
+ *
+ * // use with reactive forms
+ * <sg-input-field>
+ *   <input sgInput formControlName="...">
+ * </sg-input-field>
```
Comment (lastminutediorama) [inline src/interface/src/styleguide/input/input.stories.ts:29]: nice, thanks for both examples

File: src/interface/src/styleguide/input/input-field.component.ts [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+   */
+  @ContentChild(InputDirective, { static: false })
+  inputDirective?: InputDirective;
+
+  /**
+   * The help/error message to be displayed below the input field.
+   */
+  @Input() supportMessage = '';
+
+  /**
```
Comment (lastminutediorama) [inline src/interface/src/styleguide/input/input-field.component.ts:42]: Nit: should we clarify which icon library is currently valid for this?

**Review by lastminutediorama — APPROVED** [review]
Looks very nice, thanks for adding all of the options.

File: src/interface/src/styleguide/input/input-field.component.ts (hunk 2 above)
Comment (pflopez) [reply src/interface/src/styleguide/input/input-field.component.ts:42]: good point!

**Pull request merged**